### ISSUE TITLE
Support plugins globally for all users (/Library/Application Support/xbar/plugins/)

### ISSUE THREAD

**Issue by dstranathan** [matryer/xbar#661]
Please add support for plugins to be loaded from /Library/Application Support/xbar/plugins/...

I'm unable to get xbar to run unless my plugins are located in ~/Library/Application Support/xbar/plugins/...

I'd like to deploy an xbar plugin to all my end-users and run it regardless of who is logged into the console.

**Comment by matryer** [comment]
@dstranathan thanks for this - do you think we should just check both locations?

**Comment by magicus** [comment]
I suggest checking both location, but let user location override system location. And having some way to not only replace a system installed script, but also disable it. (I'm not sure what the disable switch in the UI is doing, but something like that.)

**Comment by dstranathan** [comment]
Yes, definitely check both locations (global and user local). User settings would trump global of course.

Thanks!

**Comment by matryer** [comment]
@leaanthony What do you think about this? It seems reasonable to me, and I think it's a relatively simple change.

**Comment by leaanthony** [comment]
Yes, I think it's a good change. We might need to indicate the location in the ui. Also, do we need an "open local plugins folder"?
